### DISCUSSION TITLE
更新日志 - 20230224

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#614] [Announcements]
# 系统：
1. 更新 r8152 驱动，以支持 RTL8156(B) 方案的 2.5G USB 有线网卡 #211 https://github.com/istoreos/istoreos/discussions/597#discussioncomment-5074576
2. 修复 Argon 主题 CSS 压缩错误导致的字体异常 #239 https://github.com/openwrt/luci/issues/5067 https://github.com/openwrt/luci/issues/5755
3. 修复 LUCI lua 版本对 ipv6 格式的判断错误 https://github.com/istoreos/istoreos/discussions/597#discussioncomment-5074077

# 应用：
* QuickStart：
  1. 修复 RAID 扩容时对 RAID 进行分区的严重 BUG（应该对加入陈列的磁盘进行分区）
  2. 修复磁盘和分区的空间使用率显示
  3. 修复部分插件安装向导状态提示
  * 以下为小伙伴 司波图 反馈 ( https://www.bilibili.com/video/BV1yA411U7Xg ) ：
  4. 修复一键配置 Aria2，qBittorrent，Transmission
  5. Aria2 增加一键配置 AriaNg 链接
  6. 修复 “内网配置” 修改 IP 以后的状态判断和自动跳转

# 设备：
1. 修复部分设备上一版本固件（20230217）内核配置变化导致的Docker容器启动失败问题

**Comment by oofyuan** [comment]
8156B的2.5G USB网卡 测试成功握手。速度很快，辛苦了！
